api/src/test/resources/org/openmrs/module/ugandaemr/include/dsdmTestDataSet.xml

```diff
@@ -67,11 +65,19 @@
     <obs  obs_id="655624" status="FINAL" person_id="1393"  concept_id="165143"  encounter_id="31182"  obs_datetime="2015-02-10 00:00:00"  location_id="2"  value_coded="165138"  creator="1"  date_created="2019-01-18 14:26:45"  voided="0" uuid="b5d9efaf-c99e-4581-b834-3d5c3448d746"/>
     <obs  obs_id="357324" status="FINAL" person_id="1393"  concept_id="99161"  encounter_id="31181" obs_datetime="2010-03-12 00:00:00"    location_id="2"  value_datetime="2013-02-06 00:00:00" creator="1"  date_created="2015-02-18 17:23:55"  voided="0" uuid="8d131f53-66b6-4552-bbd9-c178e4470b6b"/>
 
+    <program program_id="1" concept_id="160631" creator="1" date_created="2021-07-29 12:08:19" changed_by="1" date_changed="2021-07-29 12:08:19" retired="0" name="HIV Program" description="Program where HIV patients are treated and monitored" uuid="18c6d4aa-0a36-11e7-8dbb-507b9dc4c741"/>
```

Comment:
@ssmusoke this concept 160631 used in program can not be found. Introduce it in your data set.

Reply:
@slubwama Apologies I had forgotten to bring back the changes, the error should be on the logic now

Reply:
@ssmusoke did you solve this. I don't seem to see a commit that you solve this

Reply:
@slubwama my apologies I had committed but not pushed